api/openapi-spec/swagger.json

```diff
@@ -8553,7 +8553,7 @@
           "type": "array"
         },
         "topologySpreadConstraints": {
-          "description": "TopologySpreadConstraints describes how a group of pods ought to spread across topology domains. Scheduler will schedule pods in a way which abides by the constraints. This field is alpha-level and is only honored by clusters that enables the EvenPodsSpread feature. All topologySpreadConstraints are ANDed.",
+          "description": "TopologySpreadConstraints describes how a group of pods ought to spread across topology domains. Scheduler will schedule pods in a way which abides by the constraints. This field is only honored by clusters that enable the EvenPodsSpread feature. All topologySpreadConstraints are ANDed.",
```

Comment:
The tricky thing is `hack/update-openapi-spec.sh` only generates this line on a Linux machine (doesn't work on Mac). Now CI should be green.